src/renderer/services/price-api/coin-cap.js

```diff
@@ -0,0 +1,213 @@
+import got from 'got'
+import dayjs from 'dayjs'
+import moment from 'moment'
+import { min, max } from 'lodash'
+import { convert } from 'cashify'
+import { MARKET } from '@config'
+
+// All prices on the CoinCap API are standardized in USD (United States Dollar)
+const BASE_CURRENCY = 'USD'
+const AMOUNT_TO_CONVERT = 1
+
+export default class CoinCapAdapter {
+  /**
+   * Get token name from API
+   * @param  {String} token
+   * @return {(String|null)}
+   */
+  static async getTokenId (token, limit = 1000) {
+    if (this.tokenLookup) {
+      return this.tokenLookup[token.toUpperCase()]
+    }
+
+    try {
+      const uri = `${MARKET.source.coinCap}/assets?limit=${limit}`
+      const { body } = await got(uri, {
+        json: true
+      })
+      const { data } = body
```

Comment:
This is unnecessary since `data` is only used once. I'd just use `body.data.reduce...`